daemon/daemon.go

```diff
@@ -667,7 +667,15 @@ func (d *Daemon) Stop(sigCh chan<- os.Signal) error {
 
 	err := d.tomb.Wait()
 	if err != nil {
-		return err
+		// do not stop the shutdown even if the tomb errors
+		// because we already scheduled a slow shutdown and
+		// exiting here will just restart snapd (via systemd)
+		// which will lead to confusing results.
+		if restartSystem {
+			logger.Noticef("WARNING: %v", err)
```

Comment:
"WARNING: error stopping daemon: %v" perhaps?

Reply:
Thanks! Fixed and force-pushed (for easier cherry-picking into 2.39). I use: `WARNING: cannot stop daemon: %v"` hope this is also good.